Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -97,3 +97,5 @@
 --rules preferForLoop
 --rules conditionalAssignment
 --rules wrapMultilineConditionalAssignment
+--rules blankLineAfterMultilineSwitchCase
+--rules consistentSwitchStatementSpacing
```

Comment:
Will update to a new build that includes these rules once the PR is merged

Reply:
https://github.com/airbnb/swift/pull/259/commits/c1a39636d4945c6a98888f93b8dca5e853c81325